### PR TITLE
Miscellaneous documentation update

### PR DESCRIPTION
**Pull request by macisamuele**
The goal of this PR is:
 * highlight Gradle5 incompatibility of release 1.3.0
 * re-establish Gradle5+ support
 * remove all traces of `make ...` invocations (as Makefile was removed in #105 )

NOTE: Ideally we should be able to merge this once #108 troubleshooting is over.
Ideally, we should not drop support to gradle versions if we have no benefits on doing it

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
```
Comment (cortinico) [inline gradle/wrapper/gradle-wrapper.properties]: I'd revert to `5.6.2` possibly.
I'm also in favor for keeping Gradle 6.1 and just letting our users know that they need to update Gradle.

File: README.md [hunk 2]
```diff
@@ -156,15 +156,15 @@ We also recommend you set up:
 Before starting developing, please run:
 
 ```
-make install-hooks
+./gradlew installHooks
 ```
 
 This will take care of installing the pre-commit hooks to keep a consistent style in the codebase.
 
 While developing, you can run tests on the plugin using:
```
Comment (cortinico) [inline README.md]:
```suggestion
While developing, you can build, run pre-commits, checks & tests on the plugin using:
```

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (macisamuele) [reply gradle/wrapper/gradle-wrapper.properties]: I'm going to preserve 5.5 as it was the version used in 1.2.0  (https://github.com/Yelp/swagger-gradle-codegen/blob/1.2.0/gradle/wrapper/gradle-wrapper.properties#L3)

File: CHANGES.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Here you can find the release notes for this project. Please note that list of r
 * ⚙️ Gradle to 5.6.2 [#67]
 * ⚙️ Add JaCoCo for CodeCov [#66]
 
+⚠️: `swagger-gradle-codegen:plugin:1.3.0` is **NOT** supported on Gradle 5
```
Comment (cortinico) [inline CHANGES.md]: Maybe rewording to

```suggestion
⚠️: `swagger-gradle-codegen:plugin:1.3.0` is supported **ONLY** on Gradle 6+
```
